src/Be.Vlaanderen.Basisregisters.GrAr.Notifications/paket.references

```diff
@@ -1,4 +1,6 @@
 AWSSDK.SimpleNotificationService
 
+System.Text.Json
```

Comment:
@ArneD should I added it to paket.template?

Reply:
Because it's not in standard right?